src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/ReadsPipelineSpark.java

```diff
@@ -109,7 +109,7 @@ protected void runTool(final JavaSparkContext ctx) {
         final JavaRDD<GATKRead> markedFilteredReadsForBQSR = markedReads.filter(read -> bqsrReadFilter.apply(read));
```

Comment:
Here too -- you need to remove the check above:

              // TODO: workaround for known bug in List version of getParallelVariants
         if ( baseRecalibrationKnownVariants.size() > 1 ) {
             throw new GATKException("Cannot currently handle more than one known sites file, " +
                     "as getParallelVariants(List) is broken");
         }